Makefile

```diff
@@ -20,9 +20,13 @@ default : doxygen mkdocs
 
 get_irods :
 	@echo "Getting iRODS source... v[${GIT_COMMITTISH}]"
-	@if [ -d ${IRODSTARGET} ] ; then rm -rf ${IRODSTARGET} ; fi
-	@git clone https://github.com/${GITHUB_ACCOUNT}/irods ${IRODSTARGET}
+	@if [ ! -d ${IRODSTARGET} ] ; then \
+		git clone https://github.com/${GITHUB_ACCOUNT}/irods ${IRODSTARGET}; \
+		else \
+		echo "IRODSTARGET already found ... ${IRODSTARGET}"; \
+		fi
```

Comment:
This change might conflict with c066080a07ef02e45ee7f6bc9c3282399c1f072d.

Reply:
it shouldn't... that code just guarantees the file will be there... (well, technically just its parent directory, but sheesh, don't be moving these things around yourself)...

and then the `sed` will just manipulate the one line in that file.

Reply:
oh, reading again...

Reply:
The original code solved a situation in which a dev is updating documentation in irods/irods, pushing changes to their remote branch, and then rebuilding the docs. It also solved switching branches.

Your change would remove that ability.

Reply:
hmm, that is true.

the goal of this change was to be able to edit the version getting shown in the header of the rendered docs.

maybe the auto-updating with the sed line is enough...   it will always get deleted/cloned again... but then immediately be updated to be the same as GIT_COMMITTISH

Reply:
You can add a new option that enables what you want. Then it becomes a matter of passing the correct options.

Reply:
added a new commit.  resolving.